pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -39,8 +39,8 @@ type customMetricsCache struct {
 	Cache  []string
 }
 
-var customMetricsMetricsMap = make(map[string]map[string]map[string]*customMetricsCache)
-var customMetricsDimensionsMap = make(map[string]map[string]map[string]*customMetricsCache)
+var customMetricsMetricsMap = make(map[string]*customMetricsCache)
```

Comment:
Since this is a hot fix and, as we agreed we want to keep it simple, I suggest to not introduce this change here; it's not necessary for fixing the custom metrics.